odoo/tools/date_utils.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import math
+import calendar
+
+from dateutil.relativedelta import relativedelta
+
+
```

Comment:
can you add dosctrings to these methods? It's really important, since it's a generic toolbox, and (for example) it's not clear what are the type of arguments that we must pass